README.rst

```diff
@@ -53,11 +61,10 @@ Information on latexmk can be found at https://ctan.org/pkg/latexmk/.
 Documentation builds
 ^^^^^^^^^^^^^^^^^^^^
 
-There are three types of build that can be created from this codebase, they can be selected through the use of the environment variable ``CELLML_SPEC_BUILD``.
+There are two types of build that can be created from this codebase, they can be selected through the use of the environment variable ``CELLML_SPEC_BUILD``.
 
-1. Full documentation build of the normative and informative specification combined [The default build].
-2. Single page documentation build of the normative specification, set ``CELLML_SPEC_BUILD=SinglePageNormative``.
-3. Multi-paged documentation build of the normative specification, set ``CELLML_SPEC_BUILD=MultiPageNormative``.
+1. Normative and informative specification combined, the default build.
```

Comment:
I understand that if `CELLML_SPEC_BUILD` is not defined or is empty then we will build the normative+informative spec. However, because we support a value of `Normative` for `CELLML_SPEC_BUILD`, it would be good to also support a value of `NormativeAndInformative` (?) for completeness? If so, then maybe the other option ought to be `NormativeOnly`?